conjure-test/test.yml

```diff
@@ -85,28 +85,21 @@ services:
     package: com.palantir.conjure
     base-path: /test
     endpoints:
-      allOptionalQueryParams:
-        http: GET /allOptionalQueryParams
+      queryParams:
```

Comment:
why this change? no longer interesting to have the two test cases given query params just passed along?